test/e2e/v3/e2e_suite.go

```diff
@@ -31,6 +31,9 @@ import (
 	"sigs.k8s.io/kubebuilder/test/e2e/utils"
 )
 
+const certmanagerVersion = "v1.0.1"
+const certmanagerURL = "https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager-legacy.yaml"
+
```

Comment:
it should not be added here. See; https://github.com/kubernetes-sigs/kubebuilder/blob/9406d91733acdf05cbb18d96bfe4733748af1202/test/e2e/utils/test_context.go#L30

We need to update it in the utils.

Reply:
@MartinForReal ^